chapters/finale.Rmd

```diff
@@ -5,6 +5,48 @@
 > --- Terry Pratchett\index{Pratchett, Terry}
 
 We have come a long way since we first met Amira, Jun, and Sami in Section \@ref(intro-personas).
+
+Amira now has scripts, data sets and reports organized, 
+in version control, 
+and on GitHub. 
+Her work has already paid off:
+a colleague spotted a problem in an analysis step,
+that impacted five figures in one of her reports.
+Because Amira has embraced GitHub and Make,
+the colleague could easily suggest a fix through a Pull Request,
+and Amira could easily regenerate all of the affected figures.
+
+She used to be intimidated by the unix shell, 
+but now Amira finds it an essential part of her everyday work:
+she uses shell scripts to automate data processing tasks, 
+she runs her own command line tools she wrote in Python, 
+and issues countless commands to Git and Make. 
+Her comfort with the shell is also helping 
+as she learns how to run her analyses on a remote computing cluster
+from Sami.
+
+Sami had experience with Git and Make in software projects from
+their undergraduate studies, 
+but they have a new appreciation for their importance in research projects. 
+They've reworked their Git and Make workshops to use examples of data pipelines, 
+and have been getting rave reviews from participants!
+
+Sami has also gained an appreciation 
+for the importance of provenance 
+of both data and code in research. 
+They've been helping their users 
+by suggesting ways they can make their research more accessible, inspectable and reproducible. 
+
+Jun is now working on his first python package. 
+He's added better error handling so he and his
+users get better information when something goes wrong,
+he's implemented some testing strategies to give him and his users confidence his code works, and
+he's improved his documentation.
+
+Jun has also added a license, a code of conduct and contributing guidelines to his project repo. 
+He's already had a contribution that fixes some typos
+in the documentation!
```

Comment:
Again, we've mostly avoided '!' in the book - quiet café jazz rather than the flaming guitars of our youth, kind of thing.